src/Altinn.App.Core/Features/FileAnalyzis/IFileAnalyserFactory.cs

```diff
@@ -1,16 +0,0 @@
-﻿using Altinn.App.Core.Features.FileAnalysis;
-using Altinn.Platform.Storage.Interface.Models;
-
-namespace Altinn.App.Core.Features.FileAnalyzis
-{
-    /// <summary>
-    /// Interface responsible for resolving the correct file analysers to run on against a <see cref="DataType"/>.
-    /// </summary>
-    public interface IFileAnalyserFactory
```

Comment:
Can we actually delete like this? This would be a breaking change, but the PR is marked as a feature